Classes/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -102,7 +103,9 @@ public function renderNode(NodeInterface $node, ControllerContext $controllerCon
             'depth' => $node->getDepth(),
             // TODO: 'uri' =>@if.onyRenderWhenNodeIsADocument = ${q(node).is('[instanceof Neos.Neos:Document]')}
             'children' => [],
+            'matchesCurrentDimensions' => $node instanceof Node && $node->dimensionsAreMatchingTargetDimensionValues(),
```

Comment:
Checking for `$node instanceof Node` shouldn't be required here, right, since it's enforced by the argument type hint?
And even if it was, you check for `NodeInterface`, not `Node`.

Reply:
The argument type hints to `NodeInterface` and `dimensionsAreMatchingTargetDimensionValues()` is only defined in `Node`. See [`Neos\Neos\Service\ContentElementWrappingService::addCssClasses()`](https://github.com/neos/neos-development-collection/blob/master/Neos.Neos/Classes/Service/ContentElementWrappingService.php#L171).

Reply:
Ah, true, sorry.

Reply:
Seems that method should find it's way into the interface?